### ISSUE TITLE
$tyk_context reference interpreted literally when jwt claim is absent

### ISSUE THREAD

**Issue by altaurog** [TykTechnologies/tyk#2139]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
with the following in my app config:
```
  "version_data": {
    "not_versioned": false,
    "default_version": "v1",
    "versions": {
      "v1": {
        "name": "v1",
        "use_extended_paths": false,
        "global_headers": {
          "X-Username": "$tyk_context.jwt_claims_sub",
          "X-Extra-Features": "$tyk_context.jwt_claims_extf"
        }
      }
    }
  },
```
if optional `extf` claim is absent, tyk treats the context reference literally and appends the header:
`X-Extra-Features: $tyk_context.jwt_claims_extf`

**What is the expected behavior?**
tyk should omit the header or leave it empty.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
reference any non-existent `$tyk_context.jwt_claims_*` value in `global_headers` and retrieve the header in the upstream request.

**Which versions of Tyk affected by this issue? Did this work in previous versions of Tyk?**
using version from this PR: https://github.com/TykTechnologies/tyk/pull/1802

don’t know about other versions

**Comment by buger** [comment]
I think this can be already fixed in recent Tyk versions as well.

**Comment by altaurog** [comment]
okay, thanks @buger, I will try to check soon

**Comment by altaurog** [comment]
seems to be fixed in tyk v2.7.7

**Comment by altaurog** [comment]
I do see the following error message though:
```time="Apr  4 14:57:29" level=error msg="Context variable type is not supported: []interface {}"```
